zkevm-circuits/src/tx_circuit.rs

```diff
@@ -1826,6 +1836,12 @@ impl<F: Field> TxCircuit<F> {
                                     })
                             }),
                         ),
+                        (
+                            TxFieldTag::TxType,
+                            None,
+                            None,
+                            Value::known(F::from(tx.tx_type as u64)),
+                        ),                        
```

Comment:
This field should be placed after `BlockNumber` since the `TxType` tag is defined after block number.